README.md

```diff
@@ -74,6 +74,8 @@ In router's case, this step carries out some extensive dependency management usi
 $ make build
 ```
 
+Make sure to have defined `DEIS_REGISTRY` previous to this step. Otherwise, `make build` will default to `DEV_REGISTRY`. If the latter is empty, your image building will fail.
+
```

Comment:
Probably shouldn't mention `DEV_REGISTRY`.  It's "legacy" that we support that at all.  We really should just be advertising use of `DEIS_REGISTRY`.